src/server/models/listItem/providers/Lawyers.ts

```diff
@@ -19,28 +19,17 @@ export async function findPublishedLawyersPerCountry(props: {
   const offset = props.offset ?? 0;
   const countryName = props.countryName;
   const andWhere: string[] = [];
-  const jsonQuery: {
-    legalAid?: boolean;
-    proBono?: boolean;
-  } = {};
-
-  if (props.legalAid === "yes") {
-    jsonQuery.legalAid = true;
-  }
-
-  if (props.proBono === "yes") {
-    jsonQuery.proBono = true;
-  }
-
-  if (Object.keys(jsonQuery).length > 0) {
-    andWhere.push(`AND "ListItem"."jsonData" @> '${JSON.stringify(jsonQuery)}'`);
-  }
```

Comment:
We never ask the user to filter by probono or legal aid.

Reply:
Maybe we used to 🤷